contract_section/tests/test_contract.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 Tecnativa - Carlos Dauden
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Roel Adriaans - roel@road-support.nl
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestContractBase(common.SavepointCase):
```

Comment:
Maybe you can refactor the base tests for making them inheritable and avoid to rewrite each time the same base stuff. See for example https://github.com/OCA/l10n-spain/commit/b9b4c9d83fc33ba10febe37cedafc4ddf7598599 how I made that tests inheritable, and I use it in https://github.com/OCA/l10n-spain/commit/16db2446d53ed00dda08a94abb0b69f187a3f934#diff-09b5031fb79d45f262679b74a15ff361R31